packages/omg-js-childchain/src/validators/index.js

```diff
@@ -10,7 +10,8 @@ const {
 
 const childchainConstructorSchema = Joi.object({
   watcherUrl: Joi.string().required(),
-  watcherProxyUrl: Joi.string().allow('')
+  watcherProxyUrl: Joi.string().allow(''),
+  verifyingContract: validateAddress.allow('')
```

Comment:
this should be required for the childchain module or the transaction calls will fail.

Reply:
can make it `required()`, my original thought was to set it as optional as a client using the childchain module may not be calling functions that would need verifyingContract